src/help.rs

```diff
@@ -11,11 +11,13 @@ This just summarizes the functionality of this command, not the syntax of Bonnie
 
 -h, --help                                      prints this help page
 -v, --version                                   prints the current version of Bonnie
--i, --init [-t, --template <template-file>]     creates a new `bonnie.toml` configuration, potentially taking a template file to use
+-i, --init [-t, --template <template-file>]     creates a new `bonnie.toml` configuration, using the specified template file if provided.
+-e, --edit-template                             opens your Bonnie init template file in your default editor
 -c, --cache                                     caches the Bonnie configuration file to `.bonnie.cache.json` for performance (this cache must be MANUALLY updated by re-running this command!)
 
 The expected location of a Bonnie configuration file can be changed from the default `./bonnie.toml` by setting the `BONNIE_CONF` environment variable.
 The expected location of a Bonnie cache file can be changed from the default `./.bonnie.cache.json` by setting the `BONNIE_CACHE` environment variable.
+The default loocation of the Bonnie template file can be changed from `~/.bonnie/template.toml` by setting the `BONNIE_TEMPLATE` environment variable.
```

Comment:
This could be a little clearer. 
```suggestion
The expected location of your default template can be changed from the default `~/.bonnie/template.toml` by setting the `BONNIE_TEMPLATE` environment variable.
```